app/lib/pre_assembly/batch.rb

```diff
@@ -21,7 +21,6 @@ class Batch
              :progress_log_file,
              :project_name,
              :set_druid_id,
-             :stageable_discovery,
              :staging_style_symlink,
```

Comment:
unused parameter now (was only supported in the CLI anyway)

Reply:
yeee-ha!!!